synapse/lib/modelrev.py

```diff
@@ -1541,7 +1541,7 @@ async def revModel_0_2_31(self):
             await self.todos.clear()
 
             for idx, layer in enumerate(self.layers):
-                logger.debug(f'Processing references in layer {idx}')
+                logger.debug(f'Processing references in layer {idx} {layer.iden}')
```

Comment:
These really shouldn't be f-strings. They should use the args array format ( especially this once since it's in a hot loop )